README.md

```diff
@@ -82,53 +82,25 @@ compile 'com.okta.spring:okta-spring-boot-starter:${okta.springboot.version}'
 
 where ${okta.springboot.version} is the latest published version in [Maven Central](https://search.maven.org/search?q=g:com.okta.spring%20a:okta-spring-boot-starter).
 
-## Supporting client side applications - OAuth Implicit flow
+## Building API Applications - Resource Server
 
-Are you writing backend endpoints in order to support a client side application? If so follow along, otherwise skip to the next section.
+Are you building backend endpoints in order to support a client side application? If so follow along, otherwise skip to the next section.
 
 ### Configure your properties
 
 You can configure your applications properties with environment variables, system properties, or configuration files. Take a look at the [Spring Boot documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html) for more details.
 
 Only these three properties are required for a web app:
 
-| Property | Default | Details |
-|----------|---------|---------|
-| okta.oauth2.issuer     | N/A | [Authorization Server](https://developer.okta.com/docs/how-to/set-up-auth-server.html) issuer URL, i.e.: https://{yourOktaDomain}/oauth2/default |
-| okta.oauth2.clientId   | N/A | The Client Id of your Okta OIDC application |
-| okta.oauth2.clientSecret   | N/A | The Client Secret of your Okta OIDC application |
-
-There are many more properties that you can optionally configure as well. Here are some examples:
-
-| Property | Default | Details |
-|----------|---------|---------|
-| okta.oauth2.audience   | api://default | The audience of your [Authorization Server](/docs/how-to/set-up-auth-server.html) |
-| okta.oauth2.groupsClaim | groups | The claim key in the Access Token's JWT that corresponds to an array of the users groups. |
-| okta.oauth2.postLogoutRedirectUri | N/A | Set to a relative or absolute URI to enable [RP-Initiated (SSO) logout](https://developer.okta.com/blog/2020/03/27/spring-oidc-logout-options). |
```

Comment:
Moved this section below, post logout redirect config isn't needed in the resource server section